src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -97,4 +112,39 @@ export class FormDateTimePickerComponent implements ControlValueAccessor {
     get currentTimeZone(): string {
         return Intl.DateTimeFormat().resolvedOptions().timeZone;
     }
+
+    /**
+     * validates the value from the input field, if it is a date value. In case it is invalid, isInvalidDate is set to true.
+     * An empty input field is considered valid, in this case the value is assigned null
+     * @param event the change event
+     */
+    validateAndUpdateField(event: Event) {
+        const val = (event.target as HTMLInputElement).value;
+        //value is empty string in case the input field has been cleared (this is handled in updateEmptyField)
+        if (val != '') {
+            //date is InvalidDate or an actual date
+            const date = new Date(val);
+            if (dayjs(val).isValid() && this.isValidDate(date)) {
```

Comment:
```suggestion
            const date = new Date(val);
            if (dayjs(val).isValid() && this.isValidDate(date)) {
```
is there a reason why we need to do it this complicated?
1. Create a date from val
2. Check that date vor validity (i.e. check if the timestamp is numeric
3. convert it in a dayjs and check for validity

Reply:
I think we can leave out the check with dayjs